website/docs/client/loc.md

```diff
@@ -23,103 +29,232 @@ Always use the most recent state, and discard the former state.
+const draftRequest = await locsState.requestIdentityLoc({
     legalOfficer: alice,
-    description: "This is a Transaction LOC",
+    description: "This is an Identity LOC",
+    userIdentity: {
+        email: "john.doe@invalid.domain",
+        firstName: "John",
+        lastName: "Doe",
+        phoneNumber: "+1234",
+    },
+    userPostalAddress: {
+        line1: "Peace Street",
+        line2: "2nd floor",
+        postalCode: "10000",
+        city: "MyCity",
+        country: "Wonderland"
+    },
     draft: true,
+    legalFee: Lgnt.from(160),
+});
+```
+
+:::warning
+The default legal fee for an identity
+LOC is 160 LGNTs. Another value of `legalFee` should have been discussed with the LLO beforehands,
+otherwise it may reject the LOC.
+:::
+
+### Add items
+
+You may add metadata (i.e. public name-value pairs), private files and links (to other LOCs) to your LOC.
+
+Metadata can be added to, or removed from, a draft LOC:
+
+```typescript title="addMetadata"
+draftRequest = await draftRequest.addMetadata({
+    name: "Some name",
+    value: "Some value"
+});
+```
+
+```typescript title="deleteMetadata"
+draftRequest = await draftRequest.deleteMetadata({
+    name: "Some name"
 });
 ```
 
 ### Submit
-At this stage it's already possible to add [metadata](#metadata) or files, to be published later by the LO.
-When done, the request has to be submitted to the LO:
+
+When done, the request must be submitted to the LLO for review:
 
 ```typescript
 const pendingRequest = await draftRequest.submit();
 ```
 
-Then you have to wait for the LO to open the LOC:
+### After review
+
+If the LLO accepted your LOC, you can proceed and publish it:
 
 ```typescript
-const openLoc = await pendingRequest.refresh() as OpenLoc;
+const acceptedRequest = await pendingRequest.refresh() as AcceptedRequest;
+const openLoc = await acceptedRequest.open({ signer, autoPublish: true });
 ```
 
-### Metadata
+If you set `autoPublish` to false, you'll have to manually publish each item
+manually after the call to open.
 
-Metadata can be added to, or removed from, an open LOC:
+After everything has been published, just wait for the LLO to close the LOC:
 
-```typescript title="addMetadata"
-openLoc = await openLoc.addMetadata({
-    name: "Some name",
-    value: "Some value"
-});
+```typescript
+const closedLoc = await openLoc.refresh() as ClosedLoc;
 ```
 
-```typescript title="deleteMetadata"
-openLoc = await openLoc.deleteMetadata({
-    name: "Some name"
+## Transaction LOC
+
+### Request
+
+```typescript
+const draftRequest = await locsState.requestTransactionLoc({
+    legalOfficer: alice,
+    description: "This is a Transaction LOC",
+    draft: true,
+    legalFee: Lgnt.from(2000),
 });
 ```
 
-## Collection LOC
-
-### Lifecycle
+:::warning
+The default legal fee for a transaction
+LOC is 2000 LGNTs. Another value of `legalFee` should have been discussed with the LLO beforehands,
+otherwise it may reject the LOC. 
```

Comment:
```suggestion
LOC is 2000 LGNTs. Another value of `legalFee` should have been discussed with the LLO beforehand,
otherwise the LLO may reject the LOC. 
```